examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -71,6 +71,7 @@
         "TestScenesMultiFabric",
         "TestScenesFabricSceneInfo",
         "#30759: Darwin chip-tool does not support ICD registration during commissioning",
-        "TestIcdManagementCluster"
+        "TestIcdManagementCluster",
+        "Test_TC_OVENOPSTATE_2_4"
```

Comment:
Please document why this test is being excluded.  I'm 100% sure it has nothing to do with ICD registration.

Reply:
This bit should be in the "Disabled due to Events verification not supported" section above, no?

Reply:
I believe we should not disable tests - we should check in only tests that can pass.

It is somewhat difficult when test runners have different capabilties, however I am concerned if we allow some disabling then we will not know what runner can do what.

Reply:
done.